### ISSUE TITLE
SublimeLinter seems to break Emmet command

### ISSUE THREAD

**Issue by freder** [SublimeLinter/SublimeLinter#1775]
hi,

since last updating all plugins, an https://github.com/emmetio/sublime-text-plugin command shows some weird behavior.
[through some testing](https://github.com/emmetio/sublime-text-plugin/issues/134) I found out that there must be an interaction with SublimeLinter. — any idea what might be causing this?

**Comment by kaste** [comment]
Nö.

a) I don't know what the weird behavior is.  b) You don't have any SublimeLinter installed, so there isn't that much code we run here either.  c) I guess this is running on ST4 so anything can happen really.

**Comment by braver** [comment]
Also, you're using the Emmet plugin, which interferes with the normal behaviour of Sublime Text and several plugins. It does not have a track record of being a good citizen in the ecosystem. If something does not work in combination with Emmet, Emmet needs to prove it's not their fault. Then maybe we can investigate something.

**Comment by freder** [comment]
for completeness: 
- it's st3
- the 'weird behavior' can be seen in the gif in the issue I'm linking to

the emmet maintainer says he can't reproduce the issue. with that the investigation seems to have reached its end.

**Comment by kaste** [comment]
I've seen the GIF but without a description it's hard to tell.  But you're on ST3 and the latest emmet probably requires ST4.  Since it only happens by using a keybinding (shortcut) to a command, and not when invoked via the Command Palette, you hit a bug in ST3 where Sublime executes actually a *random*(**!**) command for commands using a so-called `InputHandler`.  The maintainer probably assumed you're on ST4 but should have asked maybe(?).  The repo README should maybe(?) say: ST4 only.

**Comment by braver** [comment]
There is nothing to work with here. Are you running into a conflicting key binding? We don't bind anything on `ctrl+w`. Weird stuff.

**Comment by lorenswendsen** [comment]
FWIW I also started having the same autocomplete problems that freder mentions above. It finally bothered me enough to start troubleshooting and I noticed the problem goes away if I disable SublimeLinter, which led me here.

I have Emmet and the CSS3 plugin installed. I followed the setup tweaks as as described here:
- https://packagecontrol.io/packages/CSS3

Some examples of the autocompletes I get as of 2-3 weeks ago.
padding-bo [tab] gives me
page-break-after: bo;

margin-lef [tab] gives me
max-resolution: l;

text-ali [tab]
text-emphasis: al;

I admit it could be Emmet or the CSS3 plugin to blame but just wanted to chime in that I think freder is describing a legit issue.
